process/mock/runTypeComponentsStub.go

```diff
@@ -9,36 +9,39 @@ import (
-	DataCodec                              sovereign.DataCodecHandler
-	TopicsChecker                          sovereign.TopicsCheckerHandler
-	RequestersContainerFactoryCreatorField requesterscontainer.RequesterContainerFactoryCreator
+	AdditionalStorageServiceFactory            process.AdditionalStorageServiceCreator
+	ShardCoordinatorFactory                    sharding.ShardCoordinatorFactory
+	NodesCoordinatorWithRaterFactory           nodesCoordinator.NodesCoordinatorWithRaterFactory
+	RequestHandlerFactory                      requestHandlers.RequestHandlerCreator
+	AccountCreator                             state.AccountFactory
+	OutGoingOperationsPool                     sovereignBlock.OutGoingOperationsPool
+	DataCodec                                  sovereign.DataCodecHandler
+	TopicsChecker                              sovereign.TopicsCheckerHandler
+	RequestersContainerFactoryCreatorField     requesterscontainer.RequesterContainerFactoryCreator
+	ValidatorAccountsSyncerFactoryHandlerField syncerFactory.ValidatorAccountsSyncerFactoryHandler
 }
 
 // NewRunTypeComponentsStub -
 func NewRunTypeComponentsStub() *RunTypeComponentsStub {
 	return &RunTypeComponentsStub{
-		AdditionalStorageServiceFactory:        &testscommon.AdditionalStorageServiceFactoryMock{},
-		ShardCoordinatorFactory:                sharding.NewMultiShardCoordinatorFactory(),
-		NodesCoordinatorWithRaterFactory:       nodesCoordinator.NewIndexHashedNodesCoordinatorWithRaterFactory(),
-		RequestHandlerFactory:                  requestHandlers.NewResolverRequestHandlerFactory(),
-		AccountCreator:                         &stateMock.AccountsFactoryStub{},
-		OutGoingOperationsPool:                 &sovereignMocks.OutGoingOperationsPoolMock{},
-		DataCodec:                              &sovereignMocks.DataCodecMock{},
-		TopicsChecker:                          &sovereignMocks.TopicsCheckerMock{},
-		RequestersContainerFactoryCreatorField: requesterscontainer.NewShardRequestersContainerFactoryCreator(),
+		AdditionalStorageServiceFactory:            &testscommon.AdditionalStorageServiceFactoryMock{},
+		ShardCoordinatorFactory:                    sharding.NewMultiShardCoordinatorFactory(),
+		NodesCoordinatorWithRaterFactory:           nodesCoordinator.NewIndexHashedNodesCoordinatorWithRaterFactory(),
+		RequestHandlerFactory:                      requestHandlers.NewResolverRequestHandlerFactory(),
+		AccountCreator:                             &stateMock.AccountsFactoryStub{},
+		OutGoingOperationsPool:                     &sovereignMocks.OutGoingOperationsPoolMock{},
+		DataCodec:                                  &sovereignMocks.DataCodecMock{},
+		TopicsChecker:                              &sovereignMocks.TopicsCheckerMock{},
+		RequestersContainerFactoryCreatorField:     requesterscontainer.NewShardRequestersContainerFactoryCreator(),
+		ValidatorAccountsSyncerFactoryHandlerField: syncerFactory.NewValidatorAccountsSyncerFactory(),
```

Comment:
Maybe use mock here ValidatorAccountsSyncerFactoryMock?

Reply:
This "stub" is not a stub actually, it uses mostly real comps;
Also, got the same "cyclic" import issue here, there is a todo to fix it